package MsgsDispatcherToClient;

message MsgServicesDescription {
    repeated string services = 1;
}

message MsgDispatcherToClient {
    message Content {
        optional MsgServicesDescription services_description = 1;
    }
    required Content message = 1;
}